docs/diagrams/MainLogicSequenceDiagram.puml

```diff
@@ -0,0 +1,52 @@
+@startuml MainLogicSequenceDiagram
+!include style.puml
+
+box "FridgeFriend\n" FRIDGEFRIEND_COLOR
+Participant ":FridgeFriend" as fridgefriend FRIDGEFRIEND_COLOR
+end box
+
+box Utilities UTILITIES_COLOR_T1
+Participant ":Ui" as ui UTILITIES_COLOR
+Participant ":Parser" as parser UTILITIES_COLOR
+end box
+
+box Command COMMAND_COLOR_T1
+Participant "command:Command" as command COMMAND_COLOR
+end box
+
+box Exception EXCEPTION_COLOR_T1
+Participant "e:Exception" as exception EXCEPTION_COLOR
+end box
+
+autoactivate on
+fridgefriend->fridgefriend: run()
+loop isExit
```

Comment:
Might want to clarify that is NOT on isExit then the loop continues, because it will actually quit on isExit, which might be misleading

Reply:
oh yes, thanks, good catch